ocp_resources/namespace.py

```diff
@@ -25,9 +25,14 @@ def __init__(
         teardown=True,
         label=None,
         yaml_file=None,
+        timeout=240,
```

Comment:
What is the default timeout in Resource?

Reply:
240 seconds. I'll import it from Resource to make it bond to it.